tests/tensorflow/simple-maxpool-conv2d-relu-dense.yaml

```diff
@@ -0,0 +1,22 @@
+arch: ai85net5
+dataset: simple-maxpool-con2d-dense
```

Comment:
There is no ReLu operation but name of file has Relu.
No newline at end of file.